src/Azure/Azure.Quantum.Client/generated/Models/SasUriResponse.cs

```diff
@@ -10,8 +10,8 @@
 
 namespace Microsoft.Azure.Quantum.Client.Models
 {
+    using System;
```

Comment:
this file is auto-generated from swagger, we should see if swagger can generate Uri directly but editing manually will be overwritten by next version